pandas/tests/indexing/common.py

```diff
@@ -89,7 +89,7 @@ def setup_method(self, method):
         self.series_ts_rev = Series(np.random.randn(4), index=dates_rev)
         self.frame_ts_rev = DataFrame(np.random.randn(4, 4), index=dates_rev)
 
-        self.frame_empty = DataFrame()
+        self.frame_empty = empty_frame
```

Comment:
I think you need to pass `empty_frame` as a parameter to `setup_method` here; I think would fix the CI issue you are asking about

Reply:
It looks like the common.py file can't actually access fixtures. I reverted back to DataFrame() instead.